foundations/src/telemetry/tokio/mod.rs

```diff
@@ -0,0 +1,364 @@
+//! Tokio Runtime Metrics
+//!
+//! Foundations provides a helper API for monitoring Tokio runtimes using the
```

Comment:
Add a section explaining the build configuration that it needs

Reply:
I'm going to do this at the top-level in lib.rs

Reply:
done, and left a link in here.